i18n/update_translations.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
Depending if you want to write a java application around it, yes with nashorn ;)
If not simply install `nodejs` and run `node script.js` (if I remember correctly)